apiClient.go

```diff
@@ -234,15 +234,18 @@ func (client *Client) CallRaw(method string, params interface{}) ([]byte, error)
 
 /* private */
 
-func santizeConfig(config *viper.Viper) error {
-	if config.GetString("lwApi.username") == "" {
-		return fmt.Errorf("lwApi.username is missing from config file: [%s]", config.ConfigFileUsed())
+func processConfig(config *LWAPIConfig) error {
+	if config.Username == "" {
+		return fmt.Errorf("username is missing from config")
+	}
+	if config.Password == "" {
+		return fmt.Errorf("password is missing from config file")
```

Comment:
typo? `config file` (since config is no longer a file)

Reply:
yup.  Bulk fix gone wrong.